test/powershell/Installer/UbuntuInstalled.Tests.ps1

```diff
@@ -0,0 +1,20 @@
+Describe 'Ubuntu Post-Install Tests' -Tag 'CI' {
+    BeforeAll {
+        $skip = $IsWindows
+
+        if (-not $skip)
+        {
+            $LinuxInfo = Get-Content /etc/os-release -Raw | ConvertFrom-StringData
+            $IsUbuntu = $LinuxInfo.ID -match 'ubuntu'
```

Comment:
Need to also include debian now since we always create the symlink.

Reply:
@dantraMSFT Could you please enable this test on Debian as well. Jim runs tests on all supported distros before release.

Reply:
Fixed. I also renamed the test file to be linux generic.